packages/release-mastermind/types/index.d.ts

```diff
@@ -3,0 +1,266 @@
+  | 'labels'
+  | 'stale'
+
+export interface SharedConfig {
+  ref?: string
+  enforceConventions?: EnforceConventions
+  stale?: Stale
+  labels?: {
+    [key: string]:
+      | IssueConditionConfig
+      | ProjectConditionConfig
+      | PRConditionConfig
+      | ScheduleConditionConfig
+      | SharedConditions
+  }
+}
+
+export interface SharedConventionConditions {
+  requires: number
+  conditions: Condition[] | string
+}
+export interface SharedConventionsConfig extends SharedConventionConditions {
+  failedComment: string
+  contexts?: string[]
+}
+
+export interface CreateBranch {
+  branchPrefix?: string
+  branchSuffix?: string
+  branchName: 'title' | 'short' | 'number'
+}
+
+export interface EnforceConventions {
+  onColumn?: Column[]
+  commentHeader?: string
+  commentFooter?: string
+  moveToColumn?: string
+  conventions: SharedConventionsConfig[]
+}
+
+export interface Stale {
+  staleLabel: string
+  stale?: StaleConfig
+  abandoned?: AbanondedConfig
+  conditions?: SharedConditions[]
+}
+
+export interface StaleConfig extends SharedConditions {
+  days: number
+  comment?: string
+  resolve?: string
+  commentHeader?: string
+  commentFooter?: string
+}
+
+export interface AbanondedConfig extends StaleConfig {
+  close?: boolean
+  lock?: boolean
+  label: string
+}
+
+export interface AutomaticApprove {
+  commentHeader?: string
+  commentFooter?: string
+  conventions: SharedConventionsConfig[]
+}
+
+export interface Release extends PRConditionConfig {
+  labels?: {
+    build: string
+    prerelease: string
+    patch: string
+    minor: string
+    major: string
+    breaking?: string
+  }
+  createTag?: boolean
+  createRelease?: CreateRelease
+  createMilestone?: CreateMilestone
+  createPackages?: string[] | string
+  createChangelog?: Changelog
+}
+
+export interface DuplicateHotfix {
+  prName: 'unchanged' | 'number' | string
+  titlePrefix?: string
+  branches: string[]
+}
+export interface SyncRemote {
+  localBranch: string
+  remoteBranch: string
+  localPath: string
+  remotePath: string
+  conditions: SharedConditions[]
+}
+
+export interface ReleaseChanges {
+  includeIssues?: boolean
+  sections?: Sections[]
+}
+
+export interface Sections {
+  title: string
+  body?: string
+  PRlabels: string[]
+  issueLabels?: string[]
+  includeCommitter?: boolean
+  linkPR?: boolean
+}
+
+export interface CreateRelease extends ReleaseChanges {
+  tagName?: string
+  tagPrefix?: string
+  releaseName?: string
+  releaseNamePrefix?: string
+  releaseNameSuffix?: string
+  draft?: boolean
+  prerelease?: boolean
+  useChangelog?: boolean
+}
+export interface Changelog extends ReleaseChanges {
+  title?: string
+  body?: string
+}
+
+export interface CreateMilestone {
+  milestone: 'version' | string
+  deadline?: string
+}
+
+export type Column = string | number
+
+interface AssignProject extends IssueConditionConfig {
+  owner?: string
+  user?: string
+  repo?: string
+  project: string
+  column: string
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/Videndum/action-masterminds/pullRequest?prid=7872005)
```suggestion
  column: string;
```